pandas/io/formats/printing.py

```diff
@@ -259,9 +259,10 @@ def enable_data_resource_formatter(enable: bool) -> None:
         if mimetype not in formatters:
             # define tableschema formatter
             from IPython.core.formatters import BaseFormatter
+            from traitlets import ObjectName
```

Comment:
We don't declare traitlets as a dependency as it comes from IPython?

Reply:
that's right!